python/tank/platform/engine.py

```diff
@@ -259,20 +259,17 @@ def __init__(self, tk, context, engine_instance_name, env):
         # emit an engine started event
         tk.execute_core_hook(constants.TANK_ENGINE_INIT_HOOK_NAME, engine=self)
 
-        self.log_debug("Init complete: %s" % self)
-        self.log_metric("Init")
-
-        # log the core and engine versions being used by the current user
-        util_log_user_attribute_metric("tk-core version", tk.version)
-        util_log_user_attribute_metric("%s version" % (self.name,), self.version)
-
         # if the engine supports logging metrics, begin dispatching logged metrics
         if self.metrics_dispatch_allowed:
             self._metrics_dispatcher = MetricsDispatcher(self)
             self.log_debug("Starting metrics dispatcher...")
             self._metrics_dispatcher.start()
             self.log_debug("Metrics dispatcher started.")
 
+        self.log_debug("Init complete: %s" % self)
+        # Emit an event for the software being launched
+        self.log_metric("Launched Software")
```

Comment:
this should say launched engine not launched software. multiple engines may run in a software and an engine can be shut down and spun up during software execution!

Reply:
letting @robblau answer this one. I noticed that this metric will be logged when changing context, so agree that "Launched Software" does not seem right here.